src/js/actions/layers.js

```diff
@@ -662,7 +656,7 @@ define(function (require, exports) {
         if (linkedLayers.isEmpty()) {
             return Promise.resolve();
         }
-        return this.transfer(resetBounds, document, linkedLayers, true);
+        return this.transfer(resetBounds, document, linkedLayers, true, true);
```

Comment:
for readability i would prefer there be named variables suppressHistory amendHistory which are set to true instead of just passing booleans ( this is true for a number of calls)